ocw/dataset_processor.py

```diff
@@ -391,8 +391,11 @@ def subset(target_dataset, subregion, subregion_name=None, extract=True, user_ma
     '''
 
     if not subregion.start:
-        subregion.start = target_dataset.times[0]
-        subregion.end = target_dataset.times[-1]
+        time_start = target_dataset.times[0]
```

Comment:
The helper function `_are_bounds_contained_by_dataset)_` will now always throw an exception if the `start` and `end` attributes of the subregion instance are not defined. Was there any particular reason you made this change?